pkg/apiserver/watch.go

```diff
@@ -196,7 +197,9 @@ func (s *WatchServer) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 				// client disconnect.
 				return
 			}
-			flusher.Flush()
+			if len(ch) == 0 {
```

Comment:
Do we have any guarantee on how frequently it will happen?
I know that this will happen, but if it waits for say tens of MB, that may not be good enough for us.